cmd/ctr/commands/snapshots/snapshots.go

```diff
@@ -67,6 +74,112 @@ var listCommand = cli.Command{
 	},
 }
 
+var diffCommand = cli.Command{
+	Name:      "diff",
+	Usage:     "get the diff of two snapshots. the default second snapshot is the first snapshot's parent.",
+	ArgsUsage: "[flags] <idA> [<idB>]",
+	Flags: append([]cli.Flag{
+		cli.StringFlag{
+			Name:  "media-type",
+			Usage: "media type to use for creating diff",
```

Comment:
Add a sane default here?

Reply:
+1 its good to have the default explicit, our current default is a compressed tar `application/vnd.oci.image.layer.v1.tar+gzip` through `ocispec.MediaTypeImageLayerGzip`.